lib/matplotlib/backend_bases.py

```diff
@@ -321,18 +319,17 @@ def draw_gouraud_triangles(self, gc, triangles_array, colors_array,
     def _iter_collection_raw_paths(self, master_transform, paths,
                                    all_transforms):
         """
-        Helper method (along with :meth:`_iter_collection`) to implement
-        :meth:`draw_path_collection` in a space-efficient manner.
+        Helper method (along with `_iter_collection`) to implement
+        `draw_path_collection` in a space-efficient manner.
```

Comment:
```suggestion
        `draw_path_collection` in a memory-efficient manner.
```
Perhaps? Maybe "space-efficient" has a context-specific meaning that I don't know and is fine.

Reply:
edited.